spec/support/features/doc_auth_helper.rb

```diff
@@ -49,6 +49,8 @@ def complete_doc_auth_steps_before_welcome_step(expect_accessible: false)
   end
 
   def complete_welcome_step
+    page.has_content? t('doc_auth.buttons.continue')
```

Comment:
these two additions and the `document_capture_step_helper.rb` addition seeme to make these tests run better (although not perfectly) locally -- am happy to remove though if it's adding to the noise of this larger PR

Reply:
i wonder if there is a more holistically way to solve this issue :(

Reply:
Yeah, I think based on similar discussion [here](https://github.com/18F/identity-idp/pull/12071#discussion_r2044454939) and a follow-up in [LG-16112](https://cm-jira.usa.gov/browse/LG-16112) to find a more comprehensive fix, it's preferable to back out the `page.has_content/button?` changes. 

Adding additional expectations for `have_current_path` seems good though.